src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,5 +10,8 @@ public class Messages {
     public static final String MESSAGE_INVALID_PROFILE_DISPLAYED_INDEX = "The profile index provided is invalid";
     public static final String MESSAGE_INVALID_EVENT_DISPLAYED_INDEX = "The event index provided is invalid";
     public static final String MESSAGE_PROFILES_LISTED_OVERVIEW = "%1$d profiles listed!";
-    public static final String MESSAGE_EVENTS_LISTED_OVERVIEW = "%1$d events listed!";
+    public static final String MESSAGE_EVENTS_INVALID_START_END = "Start date should be before end date!";
+    public static final String MESSAGE_EVENTS_HAS_TIME =
+            "Start date and end dates must both be either date only or include time!";
```

Comment:
Just a minor issue, the messages only mention start date and end date. I get that you are referring to the date and time, but I think it could be clearer if you mentioned the time as well. 

Or maybe something like this could work as well
"s/START should not occur after e/END"

Reply:
I agree this error message might be somewhat unclear. I think I prefer the first approach you suggested as its more in line with our other error messages. 

Maybe something like `"Start date and end dates must both be of either date only or date and time!"`. Thoughts?